src/products/breakout-deck.md

```diff
@@ -28,6 +28,7 @@ and attach the Crazyflie 2.X to it.
 
 #### Resources
 
+- {% id_link breakout-deck %}
```

Comment:
Technically the other product pages have this information on the bottom of the recourses list, but since we are probably going through a product page change anyway.. it doesn't matter that much :)

Reply:
Updated